src/uproot/interpretation/__init__.py

```diff
@@ -73,7 +73,7 @@ def awkward_form(
         context=None,
         index_format="i64",
         header=False,
-        tobject_header=True,
+        tobject_header=False,
```

Comment:
The header data that users generally don't want to see used to be obscured as field names starting with `@`, but the Forth code doesn't even read them in, so the Form (by default) should not include them, either.